src/parser/hunter/beastmastery/modules/core/GlobalCooldown.tsx

```diff
@@ -23,28 +31,27 @@ const MIN_GCD = 750;
 const MAX_GCD = 1500;
 
 /**
- * Aspect of the wild reduces Global Cooldown for damaging spells by 0.2 seconds before haste calculations
+ * Aspect of the wild reduces Global Cooldown for certain spells by 0.2 seconds
+ * before haste calculations
  */
 class GlobalCooldown extends CoreGlobalCooldown {
   static dependencies = {
     ...CoreGlobalCooldown.dependencies,
     haste: Haste,
   };
 
-  getGlobalCooldownDuration(spellId) {
+  protected haste!: Haste;
+
+  getGlobalCooldownDuration(spellId: number) {
     const gcd = super.getGlobalCooldownDuration(spellId);
     if (!gcd) {
       return 0;
     }
-    if (spellId && ASPECT_AFFECTED_ABILTIES.includes(spellId) && this.selectedCombatant.hasBuff(SPELLS.ASPECT_OF_THE_WILD.id)) {
-      let unhastedAspectGCD = MAX_GCD - ASPECT_GCD_REDUCTION;
+    if (spellId &&
```

Comment:
The parameter typing suggests this is always filled and so this check should be omitted